scio-parquet/src/test/scala/com/spotify/scio/parquet/avro/ParquetAvroIOTest.scala

```diff
@@ -79,208 +82,278 @@ class ParquetAvroIOTest extends ScioIOSpec with TapSpec with BeforeAndAfterAll {
 
   override protected def afterAll(): Unit = FileUtils.deleteDirectory(testDir)
 
-  "ParquetAvroIO" should "work with specific records" in {
+  private def createConfig(splittable: Boolean): Configuration = {
+    val c = ParquetConfiguration.empty()
+    c.set(ParquetReadConfiguration.UseSplittableDoFn, splittable.toString)
+    c
+  }
+
+  private val readConfigs =
+    Table(
+      ("config", "description"),
+      (() => createConfig(false), "legacy read"),
+      (() => createConfig(true), "splittable"),
+      (() => ParquetAvroIO.ReadParam.DefaultConfiguration, "default")
+    )
+
+  it should "work with specific records" in {
     val xs = (1 to 100).map(AvroUtils.newSpecificRecord)
-    testTap(xs)(_.saveAsParquetAvroFile(_))(".parquet")
-    testJobTest(xs)(ParquetAvroIO(_))(
-      _.parquetAvroFile[TestRecord](_).map(identity)
-    )(_.saveAsParquetAvroFile(_))
+
+    forAllCases(readConfigs) { case (c, _) =>
```

Comment:
the main idea of changes in this file is to test both SDR and Legacy in the same test cases